modules/manage/pages/cluster-maintenance/compaction-tuning.adoc

```diff
@@ -0,0 +1,48 @@
+= Optimize Compaction Settings
+:description: Redpanda's approach to compaction and options for configuring it
+
+Configure compaction for your cluster to optimize storage utilization.
```

Comment:
I'm on the fence about the notion of optimization. Compaction isn't a feature that can be turned on to optimize storage utilization in all cases. It's true, though, that it helps storage utilization when it an application's data can take advantage of the feature.